src/key_gen/tests.rs

```diff
@@ -0,0 +1,177 @@
+// Copyright 2020 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use crate::dev_utils::{create_ids, new_common_rng, PeerId, RngChoice};
+use crate::id::SecretId;
+use crate::key_gen::{message::DkgMessage, DkgPhases, KeyGen};
+use maidsafe_utilities::{serialisation::serialise, SeededRng};
```

Comment:
Let's not use this at all and use bincode direct. (we hav been trying to kill maidsafe_utils for many months now, so let's ban it's use)

Reply:
Ah, ok. will do.